easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.0.2-5-intel-2016a.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.0.2-5'
```

Comment:
@boegel : I have updated the version number, but it looks like they keep replacing versions rather then adding new versions, so this new one would also stop working after some time. It looks like the best way is to use the flag "--try-software-version"